ir/expression.def

```diff
@@ -257,6 +261,11 @@ class Constant : Literal {
 class BoolLiteral : Literal {
     bool value;
     toString{ return value ? "true" : "false"; }
+
+#emit
+    /// @return a bool literal. Both booleans are interned.
+    static const BoolLiteral *get(bool value, const Util::SourceInfo &si = {});
+#end
```

Comment:
I don't think these functions need `#emit`/`#end` wrapped around them -- should work fine without them

Reply:
Seems like this worked.